src/main/java/lania/Parser.java

```diff
@@ -0,0 +1,128 @@
+package lania;
+
+import lania.command.AddCommand;
+import lania.command.Command;
+import lania.command.CompleteCommand;
+import lania.command.DeleteCommand;
+import lania.command.ExitCommand;
+import lania.command.FindCommand;
+import lania.command.ListCommand;
+import lania.exception.LaniaEmptyDescriptionException;
+import lania.exception.LaniaException;
+import lania.task.Deadline;
+import lania.task.Event;
+import lania.task.Todo;
+
+/**
+ * This class deals with making sense of the user command
+ * by parsing the given string.
+ */
+public class Parser {
+
+    /**
+     * Decides which command to execute based on the given input.
+     *
+     * @param fullCommand The input given by user.
+     * @return The type of command to be executed.
+     */
+    public Command parse(String fullCommand) throws LaniaException {
```

Comment:
This code is fine on its own but perhaps a switch statement can help to increase the readability of the code?

Reply:
Thank you for pointing that out, I agree with you and will do so to improve the readability.